tests/ui/stable-mir-print/operands.rs

```diff
@@ -0,0 +1,25 @@
+//@ compile-flags: -Z unpretty=stable-mir --crate-type lib -C panic=abort
```

Comment:
Can you please add some examples for more aggregates. Ideas that come to mind are:
1. Structs w braced args {}
2. Structs w no args
3. Closures that capture values
4. Arrays constructed per-element (`[1, 2, 3]`) style

Reply:
BTW, for 1, this will still print `Struct ()`. But I can add more tests

Reply:
That's fine, I just want to see how the existing behavior is exercised. Better to have tests that are wrong so we can see when they become right, rather than have no tests at all for things we explicitly know act weirdly...